src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md

```diff
@@ -112,7 +114,156 @@ runnable example.
+        .UseOtlpExporter();
+    ```
+
+    For details see: [Meter](../../docs/metrics/customizing-the-sdk/README.md#meter).
+
+    When using `Microsoft.Extensions.Hosting` v8.0.0 or greater (a standard part
+    of ASP.NET Core) `Meter`s and `Instrument`s can also be enabled using
+    configuration.
+
+    `appSettings.json` metrics configuration example:
+
+    ```json
+    {
+      "Metrics": {
+        "EnabledMetrics": {
+          "Microsoft.AspNetCore.*": true,
+          "System.*": true,
+          "MyCompany.*": true,
+        }
+      }
+    }
+    ```
+
+    For details about the built-in metrics exposed by .NET see: [Built-in
+    metrics in
+    .NET](https://learn.microsoft.com/dotnet/core/diagnostics/built-in-metrics).
+
+  * Tracing
+
+    Trace telemetry is controlled by calling `TracerProviderBuilder.AddSource`
+    to listen to
+    [ActivitySource](https://learn.microsoft.com/dotnet/api/system.diagnostics.activitysource)s
+    emitting traces. Typically instrumentation packages will make this call
+    automatically.
+
+    Examples:
+
+    ```csharp
+    appBuilder.Services.AddOpenTelemetry()
+        .UseOtlpExporter()
+        .WithTracing(tracing => tracing
+            .AddSource(MyActivitySource.Name) // Listen to custom telemetry
+            .AddAspNetCoreInstrumentation() // Use instrumentation to listen to telemetry
+        );
+    ```
+
+    ```csharp
+    appBuilder.Services.ConfigureOpenTelemetryTracerProvider(tracing => tracing
+        .AddSource(MyActivitySource.Name) // Listen to custom telemetry
+        .AddAspNetCoreInstrumentation() // Use instrumentation to listen to telemetry
+    );
+
+    appBuilder.Services.AddOpenTelemetry()
+        .UseOtlpExporter();
+    ```
+
+    For details see: [Activity Source](../../docs/trace/customizing-the-sdk/README.md#activity-source).
+
+* The exporter registered by `UseOtlpExporter` will be added as the last
+  processor in the pipeline established for logging and tracing.
+
+* `UseOtlpExporter` can only be called once. Subsequent calls will result in a
+  `NotSupportedException` being thrown.
+
+* `UseOtlpExporter` cannot be called in addition to signal-specific
+  `AddOtlpExporter` methods. If `UseOtlpExporter` is called signal-specific
+  `AddOtlpExporter` calls will result in a `NotSupportedException` being thrown.
+
+### Configuring signals when using UseOtlpExporter
+
+`UseOtlpExporter` supports the full set of [environment
+variables](#environment-variables) listed below including the signal-specific
+overrides and users are encouraged to use this mechanism to configure their
+exporters.
+
+> [!NOTE]
+> In OpenTelemetry .NET environment variable keys are retrieved using
+  `IConfiguration` which means they may be set using other mechanisms such as
+  defined in appSettings.json or specified on the command-line.
+
+An `UseOtlpExporter` overload is provided which may be used to set the protocol
+and base endpoint but no other options are currently exposed:
```

Comment:
"base endpoint" is undefined and might be misleading.

Reply:
So I called it base endpoint because when using `HttpProtobuf` the signal path is added to it. There is this note on there to try and explain that:

![image](https://github.com/open-telemetry/opentelemetry-dotnet/assets/28367120/fd1d18a9-e817-44fd-9237-611817e57ad6)

It is also called `baseEndpoint` on the actual API: https://github.com/open-telemetry/opentelemetry-dotnet/blob/b21da4b074f4fa6a56dfd3c9fa27c58f28867753/src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Builder/OpenTelemetryBuilderOtlpExporterExtensions.cs#L57

Questions...

1) Did you notice that "Note" and if not, does that satisfy as an explanation?
2) If unsatisfied, do you think more docs will suffice or should the actual thing on the API be renamed?

Reply:
I guess most folks call it "base URL/URI", check https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md#endpoint-urls-for-otlphttp

![image](https://github.com/open-telemetry/opentelemetry-dotnet/assets/17327289/083979f1-d55a-4951-8b91-c7b22e5aae30)

Normally what I will do is to ask a search engine https://www.google.com/search?q=base+endpoint, and it seems the results are not leading to the right path.

Reply:
> Questions...
> 
> 1. Did you notice that "Note" and if not, does that satisfy as an explanation?
> 2. If unsatisfied, do you think more docs will suffice or should the actual thing on the API be renamed?

1. I noticed it, and I was confused by "base URL", "base endpoint" and "base address".
2. I don't know, haven't got time to think about this.

Reply:
I just pushed an update to use "base URL" instead.